### PR TITLE
Cargo crates always land at intended destination

### LINKED ISSUE #10204: crates on atlas dont land on the QM incoming crate and go straight through hitting anything on the same line
### Labels

[BUG][MINOR]

### Description

the QM belts dont catch the crates, they just keep going through them and slam into whatever is in their way

### Steps to reproduce

order crates on atlas
witness OSHA guidelines being broken in real time

### Expected Behavior

expect the crate to get caught by the belts and not be a hazard to people working

### Additional Information & Screenshots

the belts werent clogged or anything and there was nothing in the way of them

Reported by: Jckrfc03
On server: Goonstation 1 Classic: Heisenbee
Round log date: 2022-08-11-22-03
Reported on: 2022-08-11 22:20:35

### PR DESCRIPTION
**Pull request by pgmzeta**
[QOL][BALANCE]
## About the PR 

Instead of always throwing with 100 range, calculate the distance and throw with exactly enough distance to land.

When an emag is using on a QM console, crates are now thrown with 200 distance.

Makes adjustments to off-rotation maps (manta, ozymandias, horizon) in one batch to ensure they still function.

NOTE: People rushing cargo crate orders will lead to more clogs, specifically on maps don't have any exterior inbound conveyors (Clarion, Destiny). This will reduce total potential cargo volume input. 

## Why's this needed? 
proper timing between crate orders consistent between maps
crates should by default land at the delivery point, not smack into walls/people
Fixes #10204

## Changelog

```changelog
(u)glowbold
(+)Cargo orders now consistently land on intake belts.
(+)Using an EMAG on a QM supply console removes intake targeting.
```

### CODE REVIEW

File: code/modules/economy/shippingmarket.dm [hunk 1]
```diff
@@ -507,7 +507,7 @@
 					if (P && !P.density)
 						P.close()
 
-		shipped_thing.throw_at(target, 100, 1)
+		shipped_thing.throw_at(target, get_dist(spawnpoint, target), 1)
```
Comment (pgmzeta) [inline code/modules/economy/shippingmarket.dm:510]: this should maybe be stored somewhere instead of calculating it for each crate? global? feels weird.

Comment (ZeWaka) [reply code/modules/economy/shippingmarket.dm:510]: static var?

Comment (pgmzeta) [reply code/modules/economy/shippingmarket.dm]: implemented in [189ac75](https://github.com/goonstation/goonstation/pull/10780/commits/189ac752ac194d37aa5e3ecc90bd4bd59ef229ab)

**Comment by frawhst** [comment]
Can you make it so emagging any QM console makes it so crates are thrown with like 200 range for the rest of the round

**Comment by pgmzeta** [comment]
I didn't remove the special ops crate unlock from `emag_act`, so it does both that and set the throw distance right now.

**Review by frawhst — APPROVED** [review]

**Pull request merged**